plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,95 @@
+package stomp
+
+import (
+	ts "crypto/tls"
+	"net"
+
+	"github.com/go-stomp/stomp"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+//STOMP ...
+type STOMP struct {
+	Host      string `toml:"host"`
+	Username  string `toml:"username,omitempty"`
+	Password  string `toml:"password,omitempty"`
+	QueueName string `toml:"queueName"`
+	SSL       bool   `toml:"ssl"`
```

Comment:
might not need this? could determine if it's set from the certs and insecure_skip_verify

Reply:
If you ask about ssl field , it is for if i use tls connection or the tcp connection